app/models/ApprovalProcessPageReview.scala

```diff
@@ -25,7 +25,7 @@ case class ApprovalProcessPageReview(
     pageUrl: String,
     result: String = "",
     status: String = Constants.InitialPageReviewStatus,
-    comment: Option[String] = None,
+    comment: String = "",
```

Comment:
Option seems a better "option" given that you're setting the String to ""

Reply:
Updated